mods/ra/maps/fall-of-greece-1-personal-war/personal-war.lua

```diff
@@ -0,0 +1,492 @@
+--[[
+   Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+FootprintTrigger1 = { CPos.New(77, 85), CPos.New(77, 86), CPos.New(77, 87), CPos.New(77, 88) }
+Trigger1Rifles = { Rifle1, Rifle2, Rifle3, Rifle4 }
+FootprintTrigger2 = { CPos.New(90, 82), CPos.New(91, 82), CPos.New(93, 82), CPos.New(94, 82), CPos.New(95, 82), CPos.New(96, 82), CPos.New(97, 82), CPos.New(98, 82), CPos.New(99, 82), CPos.New(102, 82) }
+Grenadiers = { "e2", "e2", "e2", "e2", "e2" }
+FootprintTrigger3 = { CPos.New(88, 77), CPos.New(88, 76), CPos.New(91, 75), CPos.New(92, 75), CPos.New(93, 75), CPos.New(94, 75), CPos.New(95, 75), CPos.New(96, 75), CPos.New(97, 75), CPos.New(98, 75), CPos.New(99, 75), CPos.New(100, 75) }
+Trigger3Team = { Gren1, Gren2, Gren3, Gren4, Gren5, VillageMammoth }
+FootprintTrigger4 = { CPos.New(94, 60), CPos.New(95, 60), CPos.New(96, 60), CPos.New(97, 60), CPos.New(98, 60), CPos.New(99, 60), CPos.New(100, 60), CPos.New(101, 60), CPos.New(102, 60), CPos.New(103, 60), CPos.New(104, 60) }
+CivilianSquad = { "c1", "c1", "c2", "c2", "c3" }
+FootprintTrigger5 = { CPos.New(99, 53), CPos.New(100, 53), CPos.New(101, 53), CPos.New(102, 53), CPos.New(103, 53), CPos.New(104, 53) }
+Trigger5Team = { TeamFive1, TeamFive2, TeamFive3, TeamFive4, TeamFive5, TeamFive6, TeamFive7 }
+TentTeam = { "e1", "e1", "e1", "e1", "e3", "e3" }
+Doggos = { "dog", "dog", "dog" }
+SovietAttackers = { "3tnk", "e2", "e2", "e2", "v2rl" }
+MigEntryPath = { BaseAttackersSpawn, GuideSpawn }
+FootprintTrigger6 = { CPos.New(81, 57), CPos.New(81, 58), CPos.New(81, 59), CPos.New(81, 60), CPos.New(81, 61) }
+FootprintTrigger7 = { CPos.New(73, 58), CPos.New(73, 59), CPos.New(73, 60), CPos.New(73, 61) }
+FootprintTrigger8 = { CPos.New(51, 83), CPos.New(51, 84), CPos.New(51, 85), CPos.New(51, 86), CPos.New(51, 87), CPos.New(51, 88) }
+FootprintTrigger9 = { CPos.New(28, 77), CPos.New(29, 77), CPos.New(30, 77),  CPos.New(31, 77),  CPos.New(32, 77), CPos.New(33, 77), CPos.New(34, 77), CPos.New(35, 77), CPos.New(36, 77), CPos.New(37, 77), CPos.New(38, 77) }
```

Comment:
Maybe give them some more descriptive names?

Reply:
I can if you insist. This is a very linear mission, only FootprintTrigger9 and 10 can be swapped in order. I think it is more readable to have everything numbered as almost all of the action in this mission is caused by these footprint triggers.

Reply:
I don't insist. It was more of a suggestion. If the order is important than numerical is probably the best way.